packages/react-tracking-pixel/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "extends": "../tsconfig_base.json",
+  "compileOnSave": false,
+  "compilerOptions": {
+    "outDir": "./dist",
+    "baseUrl": "./src",
+    "rootDir": "./src"
+  },
+  "include": [
+    "../../config/typescript/**/*",
+    "./src/**/*.ts",
+    "./src/**/*.tsx"
+  ],
+  "exclude": ["**/*.test.ts", "**/*.test.tsx"],
+  "references": [{"path": "../react-html"}]
```

Comment:
We already have issues with remembering to add dependencies to `package.json` when they are local packages. We are going to want to have some way of statically enforcing that dependencies are tracked properly, whether through a linter, a custom script, a repo-level test, or whatever.

Reply:
The compiler does an ok-ish good job at checking this. If a project is missing an explicit reference here then the build fails. That said, I've seen cases of it passing without an explicit reference. I'll open up an issue.